### PR TITLE
Fix connection delete enabled

### LINKED ISSUE #2686: Buttons in Cloud Connection Still Active Although Nothing Selected
If you select a cloud coneection in the Cloud Connection Service, all the buttons are activated ("New Connection, "New pub/sub", "delete", "Connect/disconnect", "Refresh") but if you deselect a cloud connection, only "New pub/sub" and "connect/disconnect" buttons get disabled instead of all. 
So if you now press "Delete", the delete prompt shows, and you can actually click "Yes", but nothing happens. 

**Testflow:**
1. Login as admin/admin into Kura
2. Go to Cloud Connections, select a cloud connection and the deselct it
3. Observe the active buttons.

**Expected behavior**
If you deselect all the cloud connections, all the buttons (except refresh) should be disabled. 

**Screenshot No. 1:**
Screenshot 2020-03-02 at 14 37 18

**Screenshot No. 2:**
Screenshot 2020-03-02 at 14 37 23

Target Environment (please complete the following information):
Board: Raspberry Pi 3
OS version: Linux raspberrypi 4.19.75-v7+ # 1270 SMP Tue Sep 24 18:45:11 BST 2019 armv7l GNU/Linux
Additional info: /

### PR DESCRIPTION
**Pull request by MMaiero**
Closes #2686

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
Hey @MMaiero
I have checked this and have one comment - if user deselects a certain cloud connection,  the buttons are properly disabled, but the bottom window with all the parameters is still visible:
Screenshot 2020-07-07 at 08 25 02

Can you hide the bottom window when nothing is selected? I think this can be done in this PR, if you agree?

**Comment by MMaiero** [comment]
How do you get to the condition where no lines are selected? I cannot get to that point...

**Comment by LeoNerdoG** [comment]
Hey @MMaiero,
according to the codu you wrote the behaviour is that if user tries to deselct a cloud connection, first one is selected. 
Is this the behaviour you wanted? Because it is how it's manifested in the UI.

**Comment by MMaiero** [comment]
yes, matches what we talked yesterday

**Comment by LeoNerdoG** [comment]
Ok then this can be merged! 😄

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudInstancesUi.java [hunk 1]
```diff
@@ -247,6 +247,11 @@ public void setVisibility(boolean isVisible) {
     }
 
     public GwtCloudEntry getSelectedObject() {
+        GwtCloudEntry selectedCloudEntry = this.selectionModel.getSelectedObject();
```
Comment (LeoNerdoG) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudInstancesUi.java:250]: Please update header to 2020.

**Review by LeoNerdoG — CHANGES_REQUESTED** [review]
Please update the header in one of the files.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudInstancesUi.java (hunk 1 above)
Comment (MMaiero) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/cloudconnection/CloudInstancesUi.java:250]: Done

**Pull request merged**